app/javascript/article-form/components/EditorActions.jsx

```diff
@@ -31,17 +36,17 @@ export const EditorActions = ({
 
   return (
     <div className="crayons-article-form__footer">
-      <Button className="mr-2 whitespace-nowrap" onClick={onPublish}>
+      <Button
+        variant="primary"
+        className="mr-2 whitespace-nowrap"
+        onClick={onPublish}
+      >
         {published || isVersion1 ? 'Save changes' : 'Publish'}
       </Button>
 
       {!(published || isVersion1) && (
-        <Button
-          variant="secondary"
-          className="mr-2 whitespace-nowrap"
-          onClick={onSaveDraft}
-        >
-          Save <span className="hidden s:inline">draft</span>
```

Comment:
Since we don't really show it on mobile we should keep this experience the same...